c2corg_ui/templates/route/edit.html

```diff
@@ -809,7 +809,7 @@ <h2 class="heading show-phone" translate>Comments</h2>
               <textarea app-markdown-editor name="route_history"
                         ng-model="route.locales[0].route_history"
                         class="form-control"
-                        placeholder="{{'Describes here historical information about the route (date, names..) ' | translate}}"></textarea>
+                        placeholder="{{'Describe here historical information about the route (date, names..)' | translate}}"></textarea>
```

Comment:
For me "Describe historical information about the route (date, names..) here" would sound more natural.